cmd/server/accounts/validate.go

```diff
@@ -71,6 +105,20 @@ func initAccountValidation(logger log.Logger, repo Repository, strategies map[va
 			return
 		}
 
+		// TODO I would wrap it into transaction, so if strategy fails
+		// then no validation record is created
+		validation := &validator.Validation{
+			AccountID: accountID,
+			Strategy:  req.Strategy,
+			Vendor:    req.Vendor,
+			Status:    validator.StatusInit,
+		}
+		err = validations.CreateValidation(validation)
```

Comment:
Good point. Do we need to confirm the database logic will treat a missing validation row as not-validated?

Reply:
Account marked as validated only at the end of "completeAccountValidation" handler. So, if no row in db, then account is definitely not validated. I'll think how to add db tx in both init/compete handlers, but in separate PR :(

ok?

Reply:
Sure, or on the read we handle the "no rows found" case.